packages/design-system/src/components/FieldContainer/FieldContainer.jsx

```diff
@@ -128,6 +124,10 @@ FieldContainer.propTypes = {
    * Text showing the requirement ("Required", "Optional", etc.). See [Required and Optional Fields]({{root}}/guidelines/forms/#required-and-optional-fields).
    */
   requirementLabel: PropTypes.node,
+  /**
+   * A function that returns a field input element to accept render props
+   */
+  render: PropTypes.func.isRequired,
```

Comment:
I opted for using the `render` prop over `children` because <Dropdown> is a form component that also needs to use the `children` prop. By renaming to `render`, we can avoid handling [this exception in `Dropdown`](https://github.com/CMSgov/design-system/pull/899/commits/21e168ff8f350fd1964eec01e5190f3d2f14531e#diff-d2fde628dd9ee937d45834fa7105cdc0d7a00c0951aacd4b1165d4d01ea3b0ffL31)